build.gradle

```diff
@@ -10,4 +10,29 @@ buildscript {
 subprojects {
     apply plugin: 'scalafmt'
     scalafmt.configFilePath = gradle.scalafmt.config
+
+    afterEvaluate {
+        if (project.plugins.hasPlugin('application')) {
+            startScripts {
+                doLast {
+                    unixScript.text = configureUnixClasspath(unixScript)
+                }
+            }
+        }
+    }
 }
+
+def configureUnixClasspath(File script) {
+    script
+        .readLines()
+        .collect { line ->
+            // Looking for the line that starts with CLASSPATH=
+            line = line.replaceAll(~/^CLASSPATH=.*$/) { original ->
+
+            // Get original line and append it
+            // with the configuration directory.
+            original += ":\$APP_HOME/ext-lib/*:\$APP_HOME/config"
```

Comment:
Could use single quotes to circumenvent character escaping?

Reply:
Good point. This was leftover of previous attempt which was also using a variable for path separator. With that not used we can use single quote. Updated that.